.github/workflows/build-apk.yml

```diff
@@ -0,0 +1,69 @@
+name: Build APK
+
+on:
+  push:
+    branches:
+      - main
+      - feature/set-up-github-APK-button
```

Comment:
This can be removed before merging (and deleting this feature branch), if the push trigger is left